CONTRIBUTING.md

```diff
@@ -69,8 +99,8 @@ Your package must build successfully for at least x86, but we recommend trying t
 
     $ ./xbps-src -a armv7l pkg <pkgname>
 
-Runtime testing of packages and building with the `-Q` flag or with `XBPS_CHECK_PKGS=yes` set in the environment or `etc/conf` are strongly encouraged.
-New packages will not be accepted unless they have been runtime tested.
+Runtime testing of packages and building with the `-Q` flag or with `XBPS_CHECK_PKGS=yes` set in `etc/conf` is strongly encouraged.
+New packages and updates will not be accepted unless they have been runtime tested.
```

Comment:
one line says runtime testing is strongly encouraged, the other one that it's required